source/shared/cpp/ObjectModel/pch.h

```diff
@@ -50,7 +42,7 @@
 #include "Enums.h"
 #include "json/json.h"
 
-namespace AdaptiveSharedNamespace
+namespace AdaptiveCards
 {
-    constexpr const char * const c_sharedModelVersion = "1.2";
+    constexpr const char* const c_sharedModelVersion = "1.2";
```

Comment:
>1.2 [](http://example.com/codeflow?start=56&length=3)

What is this? Can we get rid of it?

Reply:
looks like this should be updated:

https://github.com/microsoft/AdaptiveCards/blob/217105e5cafeeb86ddd347c5fde0c67787c559e6/source/shared/cpp/ObjectModel/FeatureRegistration.cpp#L11-L12

Reply:
updated in 49e4d75